suite-common/wallet-core/src/transactions/__fixtures__/transactionsReducer.ts

```diff
@@ -149,3 +149,30 @@ export const addTransaction = [
         result: { [ACCOUNT.key]: [testMocks.getWalletTransaction({ txid: '00' })] },
     },
 ];
+
+export const removeTransaction = [
+    {
+        description: 'ordinary remove',
+        initialState: {
+            transactions: {
+                [ACCOUNT.key]: [testMocks.getWalletTransaction()],
+            },
+        },
+        actionPayload: {
+            account: ACCOUNT,
+            txs: [testMocks.getWalletTransaction()],
+        },
+        result: { [ACCOUNT.key]: [] },
+    },
+    {
+        description: 'remove from nonexistent account',
```

Comment:
In this case something like: `it("removing transaction from the non-existing account does nothing", ...`